core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/dot-toolbar.component.scss

```diff
@@ -35,8 +35,6 @@ p-toolbar::ng-deep {
         color: $toolbar-color;
         height: $toolbar-height;
         padding: 0 $spacing-3 0 0;
-        border: 0;
-        border-bottom: solid 1px $color-palette-gray-300;
```

Comment:
Isn't this global? I mean, are we going to remove the `border-bottom` for the dot-toolbar all over the app?

Reply:
i removed it so it can use the styles designated for a toolbar in the Design System